Cargo.toml

```diff
@@ -21,3 +21,8 @@ codegen-units = 1
 # Stripping the binary reduces the size by ~30%, but the stacktraces won't be usable anymore. 
 # This is fine as long as we don't have any unhandled panics, but let's keep it disabled for now
 # strip = true
+
```

Comment:
This profile doesn't seem used anywhere? We can replace `--release` for `--profile release-wasm` in the README instructions if it's useful